leafmap/bokehmap.py

```diff
@@ -218,14 +224,14 @@ def add_raster(
 
     def add_stac_layer(
         self,
-        url=None,
-        collection=None,
-        item=None,
-        assets=None,
-        bands=None,
-        titiler_endpoint=None,
-        attribution="",
-        fit_bounds=True,
+        url: str,
+        collection: str,
```

Comment:
The `collection` parameter is optional. It can be None. The CI error is caused by this.